model/admin/db.go

```diff
@@ -0,0 +1,93 @@
+package admin
+
+import (
+	"github.com/evergreen-ci/evergreen/db"
+	"github.com/evergreen-ci/evergreen/db/bsonutil"
+	"github.com/pkg/errors"
+	"gopkg.in/mgo.v2/bson"
+)
+
+const (
+	Collection       = "admin"
+	systemSettingsID = "0"
```

Comment:
I think we could/should have a name that is a little more explicit than 0 for this key.